src/gui/statusbar.cpp

```diff
@@ -42,225 +42,256 @@
-  layout = new QHBoxLayout(container);
-  layout->setContentsMargins(0,0,0,0);
+    Preferences* const pref = Preferences::instance();
+    connect(BitTorrent::Session::instance(), SIGNAL(speedLimitModeChanged(bool)), this, SLOT(updateAltSpeedsBtn(bool)));
+    m_container = new QWidget(bar);
+    m_layout = new QHBoxLayout(m_container);
+    m_layout->setContentsMargins(0,0,0,0);
+
+    m_container->setLayout(m_layout);
+    m_connecStatusLblIcon = new QPushButton(bar);
+    m_connecStatusLblIcon->setFlat(true);
+    m_connecStatusLblIcon->setFocusPolicy(Qt::NoFocus);
+    m_connecStatusLblIcon->setCursor(Qt::PointingHandCursor);
+    m_connecStatusLblIcon->setIcon(QIcon(":/icons/skin/firewalled.png"));
+    m_connecStatusLblIcon->setToolTip(QString::fromUtf8("<b>") + tr("Connection status:") + QString::fromUtf8("</b><br>") + QString::fromUtf8("<i>") + tr("No direct connections. This may indicate network configuration problems.") + QString::fromUtf8("</i>"));
 
-  container->setLayout(layout);
-  connecStatusLblIcon = new QPushButton(bar);
-  connecStatusLblIcon->setFlat(true);
-  connecStatusLblIcon->setFocusPolicy(Qt::NoFocus);
-  connecStatusLblIcon->setCursor(Qt::PointingHandCursor);
-  connecStatusLblIcon->setIcon(QIcon(":/icons/skin/firewalled.png"));
-  connecStatusLblIcon->setToolTip(QString::fromUtf8("<b>")+tr("Connection status:")+QString::fromUtf8("</b><br>")+QString::fromUtf8("<i>")+tr("No direct connections. This may indicate network configuration problems.")+QString::fromUtf8("</i>"));
-  dlSpeedLbl = new QPushButton(bar);
-  dlSpeedLbl->setIcon(QIcon(":/icons/skin/download.png"));
-  connect(dlSpeedLbl, SIGNAL(clicked()), this, SLOT(capDownloadSpeed()));
-  dlSpeedLbl->setFlat(true);
-  dlSpeedLbl->setFocusPolicy(Qt::NoFocus);
-  dlSpeedLbl->setCursor(Qt::PointingHandCursor);
+    m_dlSpeedLbl = new QPushButton(bar);
+    m_dlSpeedLbl->setIcon(QIcon(":/icons/skin/download.png"));
+    connect(m_dlSpeedLbl, SIGNAL(clicked()), this, SLOT(capDownloadSpeed()));
+    m_dlSpeedLbl->setFlat(true);
+    m_dlSpeedLbl->setFocusPolicy(Qt::NoFocus);
+    m_dlSpeedLbl->setCursor(Qt::PointingHandCursor);
+    m_dlSpeedLbl->setStyleSheet("text-align:left;");
 
-  upSpeedLbl = new QPushButton(bar);
-  upSpeedLbl->setIcon(QIcon(":/icons/skin/seeding.png"));
-  connect(upSpeedLbl, SIGNAL(clicked()), this, SLOT(capUploadSpeed()));
-  upSpeedLbl->setFlat(true);
-  upSpeedLbl->setFocusPolicy(Qt::NoFocus);
-  upSpeedLbl->setCursor(Qt::PointingHandCursor);
-  DHTLbl = new QLabel(tr("DHT: %1 nodes").arg(0), bar);
-  DHTLbl->setSizePolicy(QSizePolicy::Maximum, QSizePolicy::Preferred);
+    m_upSpeedLbl = new QPushButton(bar);
+    m_upSpeedLbl->setIcon(QIcon(":/icons/skin/seeding.png"));
+    connect(m_upSpeedLbl, SIGNAL(clicked()), this, SLOT(capUploadSpeed()));
+    m_upSpeedLbl->setFlat(true);
+    m_upSpeedLbl->setFocusPolicy(Qt::NoFocus);
+    m_upSpeedLbl->setCursor(Qt::PointingHandCursor);
+    m_upSpeedLbl->setStyleSheet("text-align:left;");
```

Comment:
1. By default QLabel is left-aligned. But it is not QLabel. ```m_upSpeedLbl``` is QPushButton stylized like label.
2. No.